src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -409,6 +446,43 @@ <h4 *ngIf="displaySubsection(4, 7)">Managing Session Responses</h4>
     </tm-example-box>
   </tm-instructor-help-panel>
   <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.STUDENTS_SEE_RESPONSE)"
+                            [id]="SessionsSectionQuestions.STUDENTS_SEE_RESPONSE"
+                            headerText="When do students see responses?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.STUDENTS_SEE_RESPONSE]">
+    <p>
+      Responses are available to view as soon as you hit publish on the session.
+    </p>
+  </tm-instructor-help-panel>
+  <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.EDIT_AFTER_SUBMIT)"
+                            [id]="SessionsSectionQuestions.EDIT_AFTER_SUBMIT"
+                            headerText="Can students edit responses after they have submitted them?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.EDIT_AFTER_SUBMIT]">
+    <p>
+      Students can edit responsed until you have published the responses.
+    </p>
+  </tm-instructor-help-panel>
+  <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.EXTEND_DEADLINE)"
+                            [id]="SessionsSectionQuestions.EXTEND_DEADLINE"
+                            headerText="Can I extend a sessions deadline?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.EXTEND_DEADLINE]">
+    <p>
+      It is possible to extend the deadline of a session by editing the end date to when you would like it extended to.
+    </p>
+  </tm-instructor-help-panel>
+  <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.SPECIFIC_STUDENT_EXTENSION)"
+                            [id]="SessionsSectionQuestions.SPECIFIC_STUDENT_EXTENSION"
+                            headerText="Can I extend the deadline for a specific student?">
+
+    <p>
+      At the moment it is not possible to extend the deadline for a specific student, but you can get the student to send their answers to you
+      so you can submit it for them.
```

Comment:
```suggestion
      At the moment it is not possible to extend the deadline for a specific student, but you can get the student to send their answers to you
      so that you can submit it on their behalf, using the 'moderate' feature.
```